### DISCUSSION TITLE
Synteny tracks color coding

### DISCUSSION THREAD

**Discussion by andreaminio** [GMOD/jbrowse-components#3144] [General]
Hi everyone,

I'd like ti give a different color coding to gene-based synteny tracks (gene-by-gene and synteny blocks obtained with MCscan adapter) from the pairwise alignments so they can be overlaid but still be able to distinguish them.
I'm not able to find a proper display configuration to make this work in any documentation, can you help me in this?
Moreover, I'd like to color code dotplots from the same tracks according to the match orientation...I suppose it is stil a display thing, but I cannot find a solution even for that.

Thanks in advance,

Andrea

**Comment by garrettjstevens** [comment]
For synteny, it's not currently possible to use custom coloring. You can follow this issue for updates on this feature request: https://github.com/GMOD/jbrowse-components/issues/3104. I'm not sure about dotplot, but I think it's the same.

**Reply by andreaminio** [reply]
Thanks Garrett,

I'll keep an eye on the updates

Andrea
